File: app/src/main/java/ms/imagine/foodiemate/views/ILoginView.java
package ms.imagine.foodiemate.views;

import com.google.firebase.auth.FirebaseUser;

/**
 * Created by eugen on 3/26/2018.
 */

public interface ILoginView {
    String getTag();
    void loginSuccess();
    void loginFailed();
    void onLoginStatusChanged(FirebaseUser user);
}

File: app/src/main/java/ms/imagine/foodiemate/views/ILoginView.kt
package ms.imagine.foodiemate.views

import com.google.firebase.auth.FirebaseUser

/**
 * Created by eugen on 3/26/2018.
 */

interface ILoginView {
    val tag: String
    fun loginSuccess()
    fun loginFailed()
    fun onLoginStatusChanged(user: FirebaseUser)
}
